src/main/java/Decoder/HereDecoder.java

```diff
@@ -43,6 +44,13 @@ public class HereDecoder {
 
     private static final Logger logger = LoggerFactory.getLogger(ApiRequest.class);
```

Comment:
Ist zwar nicht in diesem Pull-Request bearbeitet worden, aber der Logger wird hier falsch initialisiert. 
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(HereDecoder.class);
```